watch-library/shared/watch/watch_adc.h

```diff
@@ -27,6 +27,23 @@
 
 #include "watch.h"
 
+// matches adc.h
+#ifndef ADC_REFCTRL_REFSEL_INTREF_Val
+#define ADC_REFCTRL_REFSEL_INTREF_Val 0x0
+#endif
+
+#ifndef ADC_REFCTRL_REFSEL_INTVCC0_Val
+#define ADC_REFCTRL_REFSEL_INTVCC0_Val 0x1
+#endif
+
+#ifndef ADC_REFCTRL_REFSEL_INTVCC1_Val
+#define ADC_REFCTRL_REFSEL_INTVCC1_Val 0x2
+#endif
+
+#ifndef ADC_REFCTRL_REFSEL_INTVCC2_Val
+#define ADC_REFCTRL_REFSEL_INTVCC2_Val 0x5
+#endif
+
```

Comment:
Do the raw values of the `watch_adc_reference_voltage` need to match the `adc.h` header values or could that translation be done internally on the `hardware/` side?

Reply:
I sense this could be done internally? In fact, it probably shouldn't bother matching; if we were to port this to another microcontroller, these values almost certainly wouldn't match up with its ADC implementation either. There may also not be feature parity in terms of having these references available, but hey, one thing at a time :)